app/scripts/modules/kubernetes/src/v2/pipelines/stages/deployManifest/artifactTab.ts

```diff
@@ -0,0 +1,39 @@
+import { IComponentOptions, IController, module } from 'angular';
+import { KUBERNETES_EXECUTION_ARTIFACT_LIST } from './artifactList';
+
+import './artifactTab.less';
+
+class KubernetesExecutionArtifactTabController implements IController {
+  public stage: any;
+  public execution: any;
+}
+
+class KubernetesExecutionArtifactTabComponent implements IComponentOptions {
+  public bindings: any = { stage: '<', execution: '<' };
+  public controller: any = KubernetesExecutionArtifactTabController;
+  public controllerAs = 'ctrl';
+  public template = `
+<div class="row execution-artifacts">
+  <div class="col-md-6 artifact-list-container">
+    <h5>Consumed Artifacts</h5>
+    <div ng-if="ctrl.execution && ctrl.execution.trigger && ctrl.execution.trigger.artifacts">
+      <kubernetes-execution-artifact-list artifacts="ctrl.execution.trigger.artifacts">
```

Comment:
This isn't necessarily coming from the trigger - I would defer to the artifacts that are explicitly requested in the "req. artifacts to bind" and "manifest artifact" fields.